backend/spec/services/api_sirene_spec.rb

```diff
@@ -87,4 +87,63 @@
       expect(subject).to eq(nil)
     end
   end
+
+  describe "cache management" do
+    let(:siret) { "21920023500014" }
+    let(:memory_store) { ActiveSupport::Cache.lookup_store(:memory_store, expires_in: 514532) }
+    let(:cache) { Rails.cache }
+
+    before do
+      Timecop.freeze(Time.now)
+      allow(Rails).to receive(:cache).and_return(memory_store)
+      Rails.cache.clear
+    end
+
+    after do
+      Timecop.return
+    end
+
+    context "when service is called once" do
+      it "insee_host/token is called once" do
+        subject
+        expect(a_request(:post, "#{ENV.fetch("INSEE_HOST")}/token"))
+          .to have_been_made.once
+      end
+    end
+
+    context "when service is called twice" do
```

Comment:
either mention "10 times" instead of "twice" or use "2.times" instead of "10.times" in the test implementation.

Reply:
I've changed it 👍🏻